packages/ow-ui/src/components/PasswordInput/Password.tsx

```diff
@@ -0,0 +1,59 @@
+import { cva } from "cva"
+import { forwardRef, HTMLInputTypeAttribute, ReactComponentElement, ReactElement, useState } from "react"
+import { Label } from "@radix-ui/react-label"
+import { IconEyeCheck, IconEyeOff} from "@tabler/icons"
+
+
+
+export interface InputProps extends React.HTMLProps<HTMLInputElement> {
+    placeholder?: string
+    label?: string
+    withAsterisk?: boolean
+    error?: boolean | string
+    inputInfo?: string
+    eyeColor: "default" | "slate" | "gray"
+  }
+
+export const PasswordInput = forwardRef<HTMLInputElement, InputProps>(({label, withAsterisk, error, inputInfo, eyeColor, ...props}, ref) => {
+  const [visible,setVisibility] = useState(false)
+  const Icon = visible ? <IconEyeOff onClick={() => setVisibility(visibility => !visibility)}/> : <IconEyeCheck onClick={() => setVisibility(visibility => !visibility)}/>;
```

Comment:
Since we are passing the same props to the icon, we should instead just do a switch on the component to use
```suggestion
  const Icon = visible ? IconEyeOff : IconEyeCheck;
```